frontend/src/__tests__/cypress/cypress/tests/mocked/modelRegistry/modelVersion/modelVersionDeploy.cy.ts

```diff
@@ -262,6 +265,33 @@ const initIntercepts = ({
       { namespace: 'opendatahub' },
     ),
   );
+
+  cy.interceptK8sList(
+    TemplateModel,
+    mockK8sResourceList(
+      [
+        mockServingRuntimeTemplateK8sResource({
+          name: 'template-1',
+          displayName: 'Multi Platform',
+          platforms: [ServingRuntimePlatform.SINGLE, ServingRuntimePlatform.MULTI],
+        }),
+        mockServingRuntimeTemplateK8sResource({
+          name: 'template-2',
+          displayName: 'Caikit',
+          platforms: [ServingRuntimePlatform.SINGLE],
+          supportedModelFormats: [
+            {
+              autoSelect: true,
+              name: 'openvino_ir',
+              version: 'opset1',
+            },
+          ],
+        }),
+        mockInvalidTemplateK8sResource({}),
+      ],
+      { namespace: 'kserve-project' },
+    ),
+  );
```

Comment:
I think if you rebase, this intercept will need to move to the factored out `initDeployPrefilledModelIntercepts` util I added in https://github.com/opendatahub-io/odh-dashboard/pull/4002